src/routers/index.ts

```diff
@@ -1,9 +1,8 @@
 import { Router, Request, Response } from 'express';
-import DummyRouter from './dummyRoute';
+import UserRouter from './userRoute';
 
 const router = Router();
-const routers: Router[] = [DummyRouter];
-router.use('/api', routers);
+router.use('/api/users', UserRouter);
```

Comment:
can you format this like 

 `const routers: Router[] = [ UserRouter ]`
 `router.use('/api', routers);`
 this will help when every router will be put under that array not creatng new router, I know someone may forgett to add api and start complaining

Reply:
Done 👍